src/modules/home/components/RequestAccessBlade/RequestAccessBlade.tsx

```diff
@@ -82,6 +89,16 @@ const RequestAccessBlade: FC<RequestAccessBladeProps> = ({ onSubmit, ...bladePro
 			className={styles['c-request-access-blade']}
 		>
 			<div className="u-px-32">
+				{space && (
+					<SpacePreview
+						spaceId={space.id}
+						spaceImage={space.image || undefined}
+						spaceLogo={space.logo}
+						spaceName={space.name}
+						spaceColor={space.color || undefined}
+						spaceServiceDescription={space.serviceDescription || undefined}
```

Comment:
`space`-prefix voor deze attributen is overbodig, ofwel 1 `space` input of de typing overerven van het `space`-type in deze blade